### PR TITLE
common.xml: Clarify some OpenDroneID fields

### PR DESCRIPTION
**Pull request by friissoren**
The description for some fields were a bit misleading
and more precise information was missing in some.

Signed-off-by: Soren Friis <soren.friis@intel.com>

### CODE REVIEW

File: message_definitions/v1.0/common.xml [hunk 1]
```diff
@@ -6483,11 +6483,11 @@
...
-      <field type="uint16_t" name="direction" units="cdeg">Direction over ground (not heading, but direction of movement) in degrees * 100: 0.0 - 359.99 degrees. If unknown: 361.00 degrees.</field>
-      <field type="uint16_t" name="speed_horizontal" units="cm/s">Ground speed. Positive only. If unknown: 255.00 m/s. If speed is larger than 254.25 m/s, use 254.25 m/s.</field>
-      <field type="int16_t" name="speed_vertical" units="cm/s">The vertical speed. Up is positive. If unknown: 63.00 m/s. If speed is larger than 62.00 m/s, use 62.00 m/s.</field>
-      <field type="int32_t" name="latitude" units="degE7">Current latitude of the UA (Unmanned Aircraft). If unknown: 0 deg (both Lat/Lon).</field>
-      <field type="int32_t" name="longitude" units="degE7">Current longitude of the UA (Unmanned Aircraft). If unknown: 0 deg (both Lat/Lon).</field>
+      <field type="uint16_t" name="direction" units="cdeg">Direction over ground (not heading, but direction of movement) in degrees * 100 measured clockwise from true North: 0 - 35999 centi-degrees. If unknown: 36100 centi-degrees.</field>
```
Comment (hamishwillee) [inline message_definitions/v1.0/common.xml]: How about remove "in degrees * 100". Units are in the units field. The rest is good.

File: message_definitions/v1.0/common.xml [hunk 2]
```diff
@@ -6483,11 +6483,11 @@
...
+      <field type="uint16_t" name="speed_horizontal" units="cm/s">Ground speed. Positive only. If unknown: 25500 cm/s. If speed is larger than 25425 cm/s, use 25425 cm/s.</field>
```
Comment (hamishwillee) [inline message_definitions/v1.0/common.xml]: This is a 16 bit number, so max is 65K values. Why are we use numbers that probably were designed for an 8 bit value (ie around 25600)? Also we should juse use the last two biggest numbers right?

File: message_definitions/v1.0/common.xml [hunk 3]
```diff
@@ -6483,11 +6483,11 @@
...
+      <field type="int16_t" name="speed_vertical" units="cm/s">The vertical speed. Up is positive. If unknown: 6300 cm/s. If speed is larger than 6200 cm/s, use 6200 cm/s. If lower than -6200 cm/s, use -6200 cm/s.</field>
```
Comment (hamishwillee) [inline message_definitions/v1.0/common.xml]: Why cropped to 6300? / or why this type to store the value?

File: message_definitions/v1.0/common.xml [hunk 4]
```diff
@@ -6483,11 +6483,11 @@
...
+      <field type="int32_t" name="latitude" units="degE7">Current latitude of the UA (Unmanned Aircraft). If unknown: 0 degE7 (both Lat/Lon).</field>
```
Comment (hamishwillee) [inline message_definitions/v1.0/common.xml]: - Remove ` degE7 ` in the description.
- I'd be tempted to make "UA (Unmanned Aircraft)" to "unmanned aircraft". After all, you can't use the contraction UA elsewhere.

File: message_definitions/v1.0/common.xml [hunk 5]
```diff
@@ -6483,11 +6483,11 @@
...
+      <field type="int32_t" name="longitude" units="degE7">Current longitude of the UA (Unmanned Aircraft). If unknown: 0 degE7 (both Lat/Lon).</field>
```
Comment (hamishwillee) [inline message_definitions/v1.0/common.xml]: Remove `degE7` in the description.

File: message_definitions/v1.0/common.xml (hunk 2 above)
Comment (friissoren) [reply message_definitions/v1.0/common.xml]: The ASTM remote ID standard uses 9 bits to encode the direction and the horizontal speed.
You can see an earlier draft of the standard here: https://github.com/opendroneid/specs
Look at the Message specification.

File: message_definitions/v1.0/common.xml (hunk 3 above)
Comment (friissoren) [reply message_definitions/v1.0/common.xml]: The standard uses only 8-bit but it allows 0.5 m/s values and I didn't want to push the quantization/conversion stuff to the user of the Mavlink API so I thought it was more clear if the same unit was used for both vertical and horizontal speed.

File: message_definitions/v1.0/common.xml (hunk 1 above)
Comment (friissoren) [reply message_definitions/v1.0/common.xml]: Done

File: message_definitions/v1.0/common.xml (hunk 4 above)
Comment (friissoren) [reply message_definitions/v1.0/common.xml]: Done

File: message_definitions/v1.0/common.xml (hunk 5 above)
Comment (friissoren) [reply message_definitions/v1.0/common.xml]: Done

File: message_definitions/v1.0/common.xml (hunk 3 above)
Comment (hamishwillee) [reply message_definitions/v1.0/common.xml]: Thanks. Makes sense.

File: message_definitions/v1.0/common.xml (hunk 2 above)
Comment (hamishwillee) [reply message_definitions/v1.0/common.xml]: Fair enough. Though you

Comment (friissoren) [reply message_definitions/v1.0/common.xml]: Sorry, did your comment get cut off in half? It appears incomplete?
Do you have any other concerns or could this PR get merged?

Comment (hamishwillee) [reply message_definitions/v1.0/common.xml]: Sorry, no problem. I will merge now.

**Pull request merged**